doc/hardware/peripherals/charger.rst

```diff
@@ -0,0 +1,32 @@
+.. _charger_api:
+
+Chargers
+########
+
+The charger subsystem exposes an API to uniformly access battery charger devices. Currently,
+only reading data is supported.
+
+Note: This API is currently experimental and this doc will be significantly changed as new features
+are added to the API.
```

Comment:
```suggestion
```

already marked as experimental in the API table

Reply:
ack